internal/pikamon/commands/catch.go

```diff
@@ -74,20 +63,31 @@ func (h *Handler) catch(s *discordgo.Session, m *discordgo.MessageCreate) {
 	}
 	log.Debug("Pokemon cache still exists, attempting catch...")
 
-	// Create pokemon information object from the cache
+	// check if trainer attempting catch is registered, and return register suggestion if not
+	registered, trainerID, err := h.isRegistered(m.Author.ID)
```

Comment:
So, I think it makes more sense to just automatically register the person instead of telling them they need to register.

Reply:
Registration without specifically choosing too is kinda a no no. I'd rather have them explicitly opt in, rather than find out they are now registered on a bot without their free will.